### PR TITLE
Calculate LHE njets

### PR DESCRIPTION
**Pull request by DylanHsu**

### CODE REVIEW

File: Flat/src/JetsMods.cc [hunk 1]
```diff
@@ -574,7 +574,7 @@ void HbbSystemMod::do_execute()
   sort(btagsorted.begin(), btagsorted.end(),
        analysis.useCMVA ?
         [](const JetWrapper *x, const JetWrapper *y) { return x->base->cmva > y->base->cmva; } :
-        [](const JetWrapper *x, const JetWrapper *y) { return x->base->csv > y->base->csv; }
+        [](const JetWrapper *x, const JetWrapper *y) { return x->base->deepCSVb > y->base->deepCSVb; }
```
Comment (sidnarayanan) [inline Flat/src/JetsMods.cc:577]: I don't think you want this - you're turning on deepCSV for 2016 too. Careful with these hacks.

Comment (DylanHsu) [reply Flat/src/JetsMods.cc:577]: this is what I wanted
we will not use CSV anymore

Comment (sidnarayanan) [reply Flat/src/JetsMods.cc:577]: Even in 2016? Okay, if so, I'll merge. (Please do confirm)

Comment (DylanHsu) [reply Flat/src/JetsMods.cc:577]: yes

**Comment by sidnarayanan** [comment]
I merged this into master in bb5651f7d97590519ca844af6fcbb27fc28daef6, so I'm going to close this here.

**Pull request closed without merging**